pyproject.toml

```diff
@@ -4,7 +4,7 @@ requires = [
   'meson-python',
   'meson>=1.2.3',
 	'ninja',
-	'numpy>=1.26.0,<1.27',
+	'numpy>=2.0.0',
```

Comment:
Quick question, I don't know what this is, but does this need to be 2.0.0 or can it be 1.26.0?

Reply:
> Quick question, I don't know what this is, but does this need to be 2.0.0 or can it be 1.26.0?

These are the build dependencies. Needs to be >=2.0.0 for building, otherwise it won't run under numpy 2.0. At runtime any version >=1.23.5 is okay, as specified in the project dependencies.